### PR TITLE
opensafely user

### PR DESCRIPTION
**Pull request by bloodearnest**
- Rename jobrunner user to opensafely
- Update login banner and user messaging
- Improve user bashrc for new world

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -32,6 +32,19 @@ ln -sf /etc/opensafely/profile /etc/profile.d/opensafely.sh
...
+# add login banner
+
+ln -sf /etc/opensafely/banner /etc/update-motd.d/00-opensafely-banner
+# add our bashrc config
+if ! grep -q "etc/opensafely/bashrc" /etc/bash.bashrc; then
+    echo "test -f /etc/opensafely/bashrc && . /etc/opensafely/bashrc" >> /etc/bash.bashrc
+fi
+
+# reduce motd noise
+chmod a-x /etc/update-motd.d/{10-help-text,50-motd-news,90-updates-available,91-contract-ua-esm-status,91-release-upgrade}
```
Comment (madwort) [inline scripts/install.sh:46]: nice idea - should we keep `90-updates-available` though?

File: scripts/user.bashrc [hunk 2]
```diff
@@ -2,40 +2,35 @@
...
-Medium privacy files are at $MEDIUM_PRIVACY_STORAGE_BASE
-High privacy files are at $HIGH_PRIVACY_STORAGE_BASE
-
-To restart:
-
-    sudo systemctl restart jobrunner
-
-Please consult ~/playbook.md for operational documentation.
-EOF
+just
```
Comment (madwort) [inline scripts/user.bashrc:36]: nice

**Review by madwort — APPROVED** [review]
looks good, let's try it!

File: scripts/install.sh (hunk 1 above)
Comment (madwort) [reply scripts/install.sh:46]: (actual question - should we be doing updates more often)

Comment (bloodearnest) [reply scripts/install.sh:46]: I don't want folks to be prompted when the log in to do ad hoc this though - it should be deliberate (or automated).

**Pull request merged**